modules/bridge/src/main/ts/ephox/bridge/components/dialog/ImagePanel.ts

```diff
@@ -0,0 +1,26 @@
+import { FieldSchema, StructureSchema } from '@ephox/boulder';
+import { Optional } from '@ephox/katamari';
+
+import { FormComponent, formComponentFields, FormComponentSpec } from './FormComponent';
+
+export interface ImagePanelSpec extends FormComponentSpec {
+  type: 'imagepanel';
+  width?: string;
```

Comment:
I added it after the JIRA was written because I realised it might be necessary to put this on the image dialog as a preview feature? I guess width can be added later in a minor release if we really need it (my imagetools branch doesn't set a width).

Reply:
I thought we discussed that and said we wouldn't... though I guess it has the custom width and height that allows warping the image so it probably would be needed. I guess that's why we had the width/height on the data then in the original spec. Anyways, I'm okay with leaving it if we're okay with it potentially causing issues with responsive layouts, though in reality your only option is to change the height to resize anyways and that'll break what the developer defined.

Reply:
Lets just leave it as is I guess unless you have any strong opinions.

Reply:
I'd already removed it by the time I wrote that 😂 it really made the test complicated, but I think it's best to assume this will go with the space available on the dialog (oxide defaults it to 100% width).

if/when we use this preview on the image dialog, it'll probably need to be a tab 🤔 I guess that's a design problem!